File: Scratch-BE/Business/Models/DrawingBoardModel.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Text;

namespace Business.Models
{
    public class DrawingBoardModel
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public string Name { get; set; }
        public ChatModel Chat { get; set; }
        public List<ShapeModel> Shapes { get; set; }

		public DrawingBoardModel()
		{
			this.Chat = new ChatModel();
			this.Shapes = new List<ShapeModel>();
		}

    }
}

File: Scratch-FE/src/app/models/DrawingBoardModel.ts
import { ChatModel } from './ChatModel';
import { ShapeHelperModel } from './HelperModels/shapeHelperModel';

export class DrawingBoardModel {
	
	id : string;
	name : String;
	shapes: ShapeHelperModel[];
	chat: ChatModel;
	
	
	constructor() {
	}


}